docs/concepts.rst

```diff
@@ -83,3 +83,14 @@ The parameter list passed to the decorated function is also added to the context
        :end-before: [END transform_example_3]
 
 More details can be found at `airflow templates reference <https://airflow.apache.org/docs/apache-airflow/stable/templates-ref.html>`_
+
+.. _file_pattern:
+
+Patterns in File path
+~~~~~~~~~~~~~~~~~~~~~
+
+We also resolve the patterns in file path based on the :ref:`file_location`
+
+#. **Local** - On local we support glob pattern - https://docs.python.org/3/library/glob.html
+#. **S3** - prefix in file path - https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-prefixes.html
```

Comment:
- Resolves `File.path` using the glob standard library (https://docs.python.org/3/library/glob.html)
```suggestion
#. **S3** - Resolves `File.path` using AWS S3 prefix rules (https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-prefixes.html)
```